src/Symfony/Bridge/PhpUnit/Tests/EnumExistsMockTest.php

```diff
@@ -16,9 +16,6 @@
 use Symfony\Bridge\PhpUnit\Tests\Fixtures\ExistingEnum;
 use Symfony\Bridge\PhpUnit\Tests\Fixtures\ExistingEnumReal;
 
-/**
- * @requires PHP 8.1
```

Comment:
Should not be removed. According to the change done in the CI config, the PHPUnit bridge will require PHP 7.2+